manifests/010-prometheus-rules.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    name: samples-operator-alerts
+  name: samples-operator-alerts
+  namespace: openshift-cluster-samples-operator
+spec:
+  groups:
+  - name: SamplesOperator
```

Comment:
None of these alerts have a severity. The three severities that we use in OpenShift are:

* info: this alert never fires on its own, it only provides additional context when other alerts of a grouping are firing
* warning: this alert describes a problem that is non urgent, typically this means it's fine for someone to look at during worktime, and it usually creates a ticket in a ticket system
* critical: this needs immediate attention and would wake someone up at night, usually this means that the cluster is in a broken state

Reply:
I've added the severity label to each of them; part of next push; thanks !!